pyanaconda/modules/payloads/payload/dnf/utils.py

```diff
@@ -219,6 +221,30 @@ def get_df_map():
     return mapping
 
 
+def get_sysroot_df_map():
```

Comment:
If I'm reading it correctly this doesn't use `df` at all now.

Reply:
It returns the same output as `get_df_map`, but we cannot call `df` in this case, because the partitioning doesn't exist yet. Both functions should use the same name. What about `get_space_usage` and `get_sysroot_space_usage`?

Reply:
Fixed.